.github/workflows/reusable-deploy-api.yml

```diff
@@ -69,3 +84,18 @@ jobs:
           SENTRY_TOKEN: ${{ secrets.SENTRY_TOKEN }}
           LOGTAIL_TOKEN: ${{ secrets.LOGTAIL_TOKEN }}
           UCAN_LOG_BASIC_AUTH: ${{ secrets.UCAN_LOG_BASIC_AUTH }}
+      - name: create sentry release
+        working-directory: packages/access-api
+        run: |
+          # write ./dist/index.{js,js.map}
+          pnpm exec wrangler --env="$ENV" publish --dry-run --outdir=dist
```

Comment:
my assumption here is that the step before should already generate these as part of the wrangler action. I'd feel more comfortable here if the artefacts are build once and then sent to the places they need to go, rather than relying on the build definitely always being reproducible (it should be, but, if why tempt fate in our error reporting)

what does `SENTRY_UPLOAD` do in that previous step? is it unused, or does wrangler-action actually upload a thing to sentry (that would be surprising)

Reply:
> what does SENTRY_UPLOAD do in that previous step? is it unused, or does wrangler-action actually upload a thing to sentry (that would be surprising)

Nothing atm. (because now `wrangler.toml` doesn't define a custom build command that might read it).

Reply:
> my assumption here is that the step before should already generate these as part of the wrangler action.

Good suggestion. I had thought so too, and tried that, but iirc it didn't write the files.

Let me see if it's just because I need to pass `--outdir=dist` on the previous non-dry-run publish.

Reply:
Added a commit to only invoke `wrangler publish` once (via `wrangler-action` with custom command to pass `--outdir dist`) https://github.com/web3-storage/w3up/pull/739/commits/3af1c35b4b5158a4c45c346293f7b6b10e97dff2